BUILDING.md

```diff
@@ -12,8 +12,127 @@ For building the manual, see the README in the [Manual repository](https://githu
 
 Windows
 -------
+ Visual studio version:
+ 1.) Download:
+   https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2017
+
+   Install: Tools for Visual Studio 2019
+     MSVC v142 
+     Windows 10 SDK
+     C++ Cmake Tools for Windows
+     MSVC v141 - VS2017 C++ x64/x86 build Tools
+
+ 2.) Download:
+   http://download.qt.io/official_releases/qt/5.9/5.9.7/qt-opensource-windows-x86-5.9.7.exe
+   Install 
+      MSVC 2017 64-bit
+ 3.) Compile (Console)
+
+   set Qt5_DIR=E:\Qt\Qt5.9.7\5.9.7\msvc2017_64\lib\cmake\Qt5
+   cd {DwarfTherapist Source Dir}
+   mkdir build
+   cd bild
+   cmake -G "Visual Studio 15 2017 Win64" ..\
+   MSBuild.exe DwarfTherapist.sln /p:Configuration=Release 
+
+ 4.) Prepare DT directory
+   Copy from E:\Qt\Qt5.9.7\5.9.7\msvc2017_64\bin
+   Qt5Core.dll
+   Qt5Gui.dll
+   Qt5Network.dll
+   Qt5Qml.dll
+   Qt5Widgets.dll to build directory 
+
+   Copy Memory_layouts to build directory -> should work 
+   (Exe starts but crashes, when loading a fortress - reason unknown so far)
+###############################################################
+Cygwin version:
+ 1.) Install cygwin and select these packages:
+   cmake
+   gcc-g++
+   git
+   make
+   libQt5Quick-devel
+   libQt5Svg-devel
+   xinit
+   xorg-server
+
+ 2.) Modify files:
+
+   Modify "E:\cygwin64\usr\include\qt5\QtGui\qwindowdefs_win.h" 
+   // Maybe this is a bug in QT5 sources?
+   ....
+   #ifndef _HRESULT_DEFINED
+   #ifdef __CYGWIN__         // # ADDED
+     typedef int HRESULT;    // # ADDED
+   #else                     // # ADDED
+     typedef long HRESULT;   
+   #endif                    // # ADDED
+   #endif
+   ...
+
+ 3.) Compile 
+   cd (DwarfTherpist Source Dir)
+   mkdir build
+   cd build
+   cmake ../
+   make
+ 4.) Prepare DT directory
+   Copy Memory_layouts to 
+   ~/.local/share/Dwarf\ Therapist
+ 5.) Run DT
+   startx
+   ./DwarfTherapist.exe
+
+does connect and load a fortress 
+###############################################################
+Cygwin cross compile for windows using mingw:
```

Comment:
I use MSYS2 for mingw builds, it is very similar but it does not require the Toolchain-mingw32.cmake file to be added as cmake already have the proper generators.